src/FailureDetectorConfig.md

```diff
@@ -85,3 +88,59 @@ config.setProperty("hazelcast.heartbeat.phiaccrual.failuredetector.sample.size",
 config.setProperty("hazelcast.heartbeat.phiaccrual.failuredetector.min.std.dev.millis", "100");
 [...]
 ```
+
+### Ping Failure Detector
+
+This failure detector is based on `InetAddress.isReachable()`.
+When the JVM process has enough permissions to create RAW sockets, the implementation will choose to rely on ICMP Echo requests, whereas, if not enough permissions, it will fallback on attempting a TCP Echo on port 7. In the latter, both a successful connection or an explicit rejection, will be treated as `Host is Reachable`.
+
+For the Ping Failure Detector to rely **only** on ICMP Echo requests, there are some criteria that need to be met.
+#### Requirements
+- Supported OS, as of Java 1.8 *nix environment support this
+- The Java executable must have the `cap_net_raw` capability. To do so, run `sudo setcap cap_net_raw=+ep path_to_java_exec`
```

Comment:
I think some people can interpret Java executable as an executable JAR or at least, it can confuse them. Let's add an example to make it clear. Add at the end of the sentence, ", for example 'sudo setcap cap_net_raw=+ep /usr/java/latest/bin/java'

Reply:
Changed to <JDK_HOME>